src/tracking/device-id.ts

```diff
@@ -0,0 +1,61 @@
+import { v4 as uuid } from "uuid";
```

Comment:
I think we should use v7 here. It has much better locality for indexing and is still sufficiently unique.

https://www.cybertec-postgresql.com/en/unexpected-downsides-of-uuid-keys-in-postgresql/

Reply:
Looks like the library doesn't implement v7 yet :( https://www.npmjs.com/package/uuid